draft-js-side-toolbar-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "draft-js-side-toolbar-plugin",
-  "version": "3.0.2",
+  "version": "3.0.3",
```

Comment:
Do not increase version in this PR. We will do that separately.

Reply:
Sure, sorry about it